solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -105,11 +105,11 @@ contract RandomBeacon is IRandomBeacon, Ownable {
     /// @notice Reward in T for notifying the operator is ineligible.
     uint256 public ineligibleOperatorNotifierReward;
 
-    /// @notice Slashing amount for supporting malicious DKG result. Every
+    /// @notice Slashing amount for submitting a malicious DKG result. Every
     ///         DKG result submitted can be challenged for the time of
     ///         `dkgResultChallengePeriodLength`. If the DKG result submitted
```

Comment:
According to your comment https://github.com/keep-network/keep-core/pull/2751#discussion_r816875050 I think we should change it like this:
```suggestion
    ///         `resultChallengePeriodLength`. If the DKG result submitted
```

Reply:
I changed that in [7c7224d](https://github.com/keep-network/keep-core/pull/2751/commits/7c7224d10cab82d7d2ed811c45bbc2b0123e7a17).